lib/bokken/uploaders/document.ex

```diff
@@ -4,8 +4,18 @@ defmodule Bokken.Uploaders.Document do
   """
   use Waffle.Definition
   use Waffle.Ecto.Definition
+  alias Bokken.Documents.File
+  @extension_whitelist ~w(.jpg .jpeg .gif .png)
```

Comment:
We need to allow more extensions to be saved. In fact I think it would be easier to ban certain extensions (anything that is an executable file for example) then it would be to have a huge list of allowed extensions